src/OrchardCore.Modules/OrchardCore.ContentFields/ViewModels/EditContentPickerFieldViewModel.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Metadata.Models;
+
+namespace OrchardCore.ContentFields.ViewModels
+{
+    public class EditContentPickerFieldViewModel
+    {
+        public string ContentItemIds { get; set; }
+        public ContentPickerField Field { get; set; }
+        public ContentPart Part { get; set; }
+        public ContentPartFieldDefinition PartFieldDefinition { get; set; }
+
+        [BindNever]
+        public IList<ContentPickerItemViewModel> SelectedItems { get; set; }
```

Comment:
This could be using a SelectList instead of a custom class. The SelectList has already other params that could be usefull for other types of frontend controls.

Reply:
Wouah, this component is magical! IT doesn't add much more to what is already done, but this might be a default solution for most of our needs.

Reply:
I just saw a SelectListItem in O1, and I think we can't use it, the Text is a string and not localizable.